crates/holochain/src/test_utils/wait_for.rs

```diff
@@ -2,11 +2,12 @@
 use std::time::Duration;
 
 #[macro_export]
-macro_rules! wait_for_any {
+macro_rules! wait_for {
     ($wait:expr, $test:expr, $check:expr, $assert:expr) => {{
+        let mut w = $wait;
         loop {
             let o = $test;
-            if !$wait.wait_any().await || $check(&o) {
```

Comment:
Oh that's an interesting issue, was this causing issues when `expr` was a function?

Reply:
Well, this wasn't even being used anywhere. It just seemed onerous and too-opinionated to use the way it was written so I rewrote it.